### ISSUE TITLE
SD2Vita and mem card unmount after sleeping for longer time.

### ISSUE THREAD

**Issue by Urbs97** [CelesteBlue-dev/PSVita-StorageMgr#51]
I'm mounting the sd2vita on ux0 and the mem card on uma0. (on a Vita OLED)

Like the title says after having my Vita in sleep mode for longer time both cards stop working.
The game or program will still work until i want to save or it needs to load something.
And if i get out and try to launch an app the Vita will throw error codes.

I'm sure that the SD2Vita unmounts but i also believe the mem card does since psp games on uma0 won't work either.

Only unplugging and plugging the SD2Vita again with a few restarts fixes this issue until it happens again.

**Comment by windsurfer1122** [comment]
Have you made sure to use the latest release 3.0, as several wake up issues have been fixed there.

**Comment by Urbs97** [comment]
Yes i'm using the latest version and don't have any problems with wake ups.
The problem is the unmounting of my cards.

**Comment by windsurfer1122** [comment]
Will setup my Fat Vita like this with SD 128GB and MemCard 64GB, and will check if it happens to me too.

**Comment by Urbs97** [comment]
Okay thank you very much!
My Fat Vita is on 3.60 enso btw, forgot to mention it but don't know how important this is.

**Comment by Urbs97** [comment]
It's even under one hour for this issue on my setup.
I'm also traveling with my Vita, so it's probably the adapter then.
Gonna buy a new one and just pray it will work.

**Comment by Urbs97** [comment]
I have it inside of an case and usually in my backpack.

**Comment by windsurfer1122** [comment]
Yesterday, I had the whole day in my backpack, drove to another, carried the whole over an exhibition and drove back. Checked every 3 hours but no issues at all.
I would really assume an issue with SD2Vita hardware itself and get a new one. Complain at your dealer. If you are in the EU you can also try srident.com

**Comment by pagina123** [comment]
I just bought my vita, I switched it off, after putting it on again I discovered that the game bubbles and app has disappeared don't know how to recover this

**Comment by windsurfer1122** [comment]
@pagina123: Off-topic

**Comment by pagina123** [comment]
Yes, about 2 months ago. Just guessing if that can be the cause
